harvard-lts/CURIOSity
# Rolling back the gallery plugin

**Rolling gallery back.**
* * *


# What does this Pull Request do?
Gallery 4 messed up a view so we're returning to gallery 3

# How should this be tested?

Spin this up locally, and go to a collection and check the gallery and masonry views. Compare against QA (which is wrong) and you'll see they look right on your local.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
